FirebaseRemoteConfig/Sources/FIRRemoteConfig.m

```diff
@@ -43,6 +43,8 @@
 static NSString *const kRemoteConfigMinimumFetchIntervalKey = @"_rcn_minimum_fetch_interval";
 /// Timeout value for waiting on a fetch response.
 static NSString *const kRemoteConfigFetchTimeoutKey = @"_rcn_fetch_timeout";
+/// Notification when config is successfully activated
+const NSNotificationName FIRRemoteConfigChangeNotification = @"FIRRemoteConfigChangeNotification";
```

Comment:
Minor (naming): could/should this be `FIRRemoteConfigActivateNotification` to disambiguate from realtime's `onConfigUpdateListener` naming?

Reply:
Good call. Done.